src/Helper/Navigation/PluginManager.php

```diff
@@ -67,20 +66,19 @@ class PluginManager extends HelperPluginManager
     ];
 
     /**
-     * @param null|ConfigInterface|ContainerInterface $configOrContainerInstance
-     * @param array $v3config If $configOrContainerInstance is a container, this
-     *     value will be passed to the parent constructor.
+     * @param ContainerInterface $configOrContainerInstance
+     * @param array $v3config
      * @psalm-param ServiceManagerConfiguration $v3config
      */
     public function __construct($configOrContainerInstance = null, array $v3config = [])
     {
-        /** @psalm-suppress MissingClosureParamType */
+        /** @psalm-suppress UnusedClosureParam, MissingClosureParamType */
```

Comment:
Note: one way to instruct psalm about unused parameters (which are still necessary) is to name them `$_something`, where `$_` indicates the intent of leaving them unused

Reply:
Thanks @Ocramius, I didn't know that - The coding standard won't allow it though!